LandSandBoat/server
# Migrate Abjurations from Magic Numbers to items.lua

**_I affirm:_**

## What does this pull request do?

Breaks out Abjuration Table into its own global
Migrates magic numbers to named entries in items.lua
Improves readability of abjurations
Audits and fixes typos/mistakes introduced in 0826ffd6a5b1fdc5109baf036493ab69e2dbb67d

replaces #2277 (Thanks @Hinekri!)

## Steps to test these changes

Log in a test character, add some abjuration materials, trade them to Alphollon C Meriard, check expected results.

![image](https://user-images.githubusercontent.com/3996176/178991951-12a0102b-43f6-4280-8ee5-1a7790cf0637.png)
